test/form/samples/pure-comment-line-break/_expected.js

```diff
@@ -6,17 +6,17 @@ console.log('should remain impure');
 console.log('code');
 console.log('should remain impure');
 
-console.log('code');
+console.log('code')/*@__PURE__*/;
 console.log('should remain impure');
 console.log('should remain impure');
 
-console.log('code');
+console.log('code');/*@__PURE__*///@__PURE__
```

Comment:
This looks wrong

Reply:
Fixed in latest PR by parsing the gap between the pure comment and the call with `acorn.parse` instead of just relying on whitespace being there. If it's just whitespace or comments, then there will be zero statements.